iothub/service/src/IotHubConnectionProperties.cs

```diff
@@ -35,6 +35,12 @@ protected IotHubConnectionProperties(string hostName)
             IotHubName = GetIotHubName(hostName);
             AmqpEndpoint = new UriBuilder(CommonConstants.AmqpsScheme, HostName, AmqpConstants.DefaultSecurePort).Uri;
             HttpsEndpoint = new UriBuilder(HttpsEndpointPrefix, HostName).Uri;
+#if NET451
+            AmqpAudience = new string[] { };
+
```

Comment:
extra newline here

Reply:
I ended up making this code better by using lists :) I'm not a fan of these #if statements.